stm32cube/stm32h7xx/CMakeLists.txt

```diff
@@ -19,16 +19,19 @@ zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DAC drivers/src/stm32h7xx_hal_
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DAC_EX drivers/src/stm32h7xx_hal_dac_ex.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DCMI drivers/src/stm32h7xx_hal_dcmi.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DFSDM drivers/src/stm32h7xx_hal_dfsdm.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DFSDM_EX drivers/src/stm32h7xx_hal_dfsdm_ex.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DMA drivers/src/stm32h7xx_hal_dma.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DMA2D drivers/src/stm32h7xx_hal_dma2d.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DMA_EX drivers/src/stm32h7xx_hal_dma_ex.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DSI drivers/src/stm32h7xx_hal_dsi.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DTS drivers/src/stm32h7xx_hal_dts.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_ETH drivers/src/stm32h7xx_hal_eth.c)
-zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_ETH_EXT drivers/src/stm32h7xx_hal_eth_ex.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_ETH_EX drivers/src/stm32h7xx_hal_eth_ex.c)
```

Comment:
This symbol does not exist on Zephyr side. It has to be defined in modules/Kconfig.stm32.
(CONFIG_USE_STM32_HAL_ETH_EXT was not defined either)

Reply:
this is added in modules/Kconfig.stm32 with https://github.com/zephyrproject-rtos/zephyr/pull/23259